Workbooks/Usage/Analysis of Page Views/Analysis of Page Views.workbook

```diff
@@ -1,22 +1,17 @@
 {
-  "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json",
   "version": "Notebook/1.0",
-  "isLocked": true,
   "items": [
     {
       "type": 1,
       "content": {
         "json": "# Analysis of Page Views\nPage views correspond to user activity in your app. Understanding how your users interact with your pages will give you good insights into what is working in your app and what aspects need improvements. \n\nThis report will help you understand common page view behaviors like:\n* Usage\n* Time spent on page\n* Time to first interaction\n* Exit rates \n\nIf your telemetry does not include page views, then follow these [instructions](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-javascript) to begin collecting -- it is relatively straightforward."
       },
-      "conditionalVisibility": null,
-      "halfWidth": false
+      "name": "text - 0"
```

Comment:
Maybe change these step names to something useful since you're adding them anyways?

Reply:
Thanks - added readable step names to all controls.